pip/commands/run.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from pip.basecommand import Command, SUCCESS
+
+from pip._vendor import rwt
+
+
+class RunCommand(Command):
+    """Show help for commands"""
+    name = 'run'
+    usage = rwt.commands.help_doc
```

Comment:
Possibly this does enough to generate docs, but can you check locally? I'll try to double-check by downloading a local copy of the PR myself, but I don't have the time to do that right now.